client/src/apps/match/MatchApp.tsx

```diff
@@ -1,9 +1,40 @@
+{/*This is used for testing. it will be deleted later */}
+// import {matchApp} from "server/src/Schema";
```

Comment:
Might want to remove these lines, the imports are not for testing and we don't need matchApp anymore